frankDoc/src/main/java/nl/nn/adapterframework/frankdoc/model/FrankDocModel.java

```diff
@@ -77,9 +77,11 @@
 	private final ElementRole.Factory elementRoleFactory = new ElementRole.Factory();
 	private Map<Set<ElementRole.Key>, ElementRoleSet> allElementRoleSets = new HashMap<>();
 	private AttributeEnumFactory attributeEnumFactory = new AttributeEnumFactory();
+	private @Getter String rootClassName;
 
-	FrankDocModel(FrankClassRepository classRepository) {
+	FrankDocModel(FrankClassRepository classRepository, String rootClassName) {
 		this.classRepository = classRepository;
+		this.rootClassName = rootClassName;
 	}
 
 	/**
```

Comment:
Is het commentaar op regel 88-90 nog relevant?

Reply:
Nee, niet meer. Ik heb het weggehaald.